cloudimg/aws.py

```diff
@@ -413,14 +416,28 @@ def upload_to_container(self, image_path, container_name, object_name,
         if image_path.lower().startswith('http'):
             # Stream the upload from a remote URL
             log.info('Opening stream to: %s', image_path)
-            resp = requests.get(image_path, stream=True)
+            resp = requests.get(image_path, stream=True, timeout=30)
```

Comment:
I assume it was added to make bandit happy :-)
Please add it to separate commit in the PR

Reply:
Yeah, will adjust.